gammapy/background/models.py

```diff
@@ -71,3 +81,775 @@ def evaluate(self, x, y):
+                                   energy_bins=energy_bins,
+                                   background=background)
+
+    @staticmethod
+    def from_fits_image(tbhdu):
+        """Read cube background model from a fits image.
+
+        Parameters
+        ----------
+        tbhdu : `~astropy.io.fits.BinTableHDU`
+            HDU binary table for the bg cube
+
+        Returns
+        -------
+        bg_cube : `~gammapy.models.CubeBackgroundModel`
+            bg model cube object
+        """
+        raise NotImplementedError
+
+    @staticmethod
+    def read(filename, format='bin_table'):
+        """Read cube background model from fits file.
+
+        Several input formats are accepted, depending on the value
+        of the 'format' parameter:
+
+        * bin_table (default and preferred format): `~astropy.io.fits.BinTableHDU`
+        * image (alternative format): `~astropy.io.fits.ImageHDU`
+
+        Parameters
+        ----------
+        filename : `~string`
+            name of file with the bg cube
+        format : `~string`, optional
+            format of the bg cube to read
+
+        Returns
+        -------
+        bg_cube : `~gammapy.models.CubeBackgroundModel`
+            bg model cube object
+        """
+        hdu = fits.open(filename)
+        if format == 'bin_table':
+            return CubeBackgroundModel.from_fits_bin_table(hdu)
+        elif format == 'image':
+            return CubeBackgroundModel.from_fits_image(hdu)
+        else:
+            raise ValueError("Got format {}, which is not understood."
+                             .format(format))
+
+    def to_astropy_table(self):
+        """Convert cube background model to astropy table format.
+
+        The name of the table is stored in the table meta information
+        under the keyword 'name'.
+
+        Returns
+        -------
+        table : `~astropy..table.Table`
+            table containing the bg cube
+        """
+        # fits unit string
+        u_detx = '{0.unit:FITS}'.format(self.detx_bins)
+        u_dety = '{0.unit:FITS}'.format(self.dety_bins)
+        u_energy = '{0.unit:FITS}'.format(self.energy_bins)
+        u_bg = '{0.unit:FITS}'.format(self.background)
+
+        # data arrays
+        a_detx_lo = Quantity([self.detx_bins[:-1]])
+        a_detx_hi = Quantity([self.detx_bins[1:]])
+        a_dety_lo = Quantity([self.dety_bins[:-1]])
+        a_dety_hi = Quantity([self.dety_bins[1:]])
+        a_energy_lo = Quantity([self.energy_bins[:-1]])
+        a_energy_hi = Quantity([self.energy_bins[1:]])
+        a_bg = Quantity([self.background])
+
+        # table
+        table = Table()
+        table['DETX_LO'] = a_detx_lo
+        table['DETX_HI'] = a_detx_hi
+        table['DETY_LO'] = a_dety_lo
+        table['DETY_HI'] = a_dety_hi
+        table['ENERG_LO'] = a_energy_lo
+        table['ENERG_HI'] = a_energy_hi
+        table['Bgd'] = a_bg
+
+        table.meta['E_THRES'] = a_energy_lo.flatten()[0].value
```

Comment:
This is the lowest energy bin, no?
This is usually much below the "energy threshold", i.e. the "safe energy threshold".

Do the background cubes from @mimayer have energy threshold defined in the FITS header?
If yes, you should store it as an extra data member.
If no, don't worry and just don't set a `E_THRES` here.